lms/djangoapps/certificates/migrations/0023_default_modes.py

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+from django.core.files import File
+from django.conf import settings
+
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        """Add default modes"""
+        for mode in ['honor', 'verified', 'professional']:
```

Comment:
Would be good to mark one of the badges as the default one, to avoid failing badge creation for other modes - honor is the lowest certification level, so out of the three available now that would likely be the one. Could be good to replace it with a more generic one later on though -- cc @pbaruah